### PR TITLE
Remove v1 and add Content type to PUT and POST

### PR DESCRIPTION
**Pull request by laurenfrederick**

### CODE REVIEW

File: content/version.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ To use any version, include the version number in the path before the query endp
 
 #### Example Request
 ```curl
-$ curl https://example.com/v1/version
+$ curl https://example.com/version
 ```
 
 #### Example Response
```
Comment (kkelly51) [inline content/version.md]: And you left the v1 in the response on purpose?

File: content/auth.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ POST /v1/refresh
 #### Example request
 
 ```curl
-$ curl --request POST https://example.com/v1/refresh --data '{"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhY2Nlc3NUb2tlbiI6IjIyMzc0YWE4MDM1M2E3ODFkYWJjYmFhZGJhOGE3ZmMwZmE1MWYzYjQzNWYxNTc4MjU2NjA0ZjFiNGQ0NTE2ODYiLCJleHAiOiIxNTQ0NDY1MDk3ODczIn0.SxFtZ7dqp9KsUSn1uTXhWis8Il8Hig8mwLANGU3cXhY"}'
+$ curl --request POST https://example.com/refresh --data '{"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhY2Nlc3NUb2tlbiI6IjIyMzc0YWE4MDM1M2E3ODFkYWJjYmFhZGJhOGE3ZmMwZmE1MWYzYjQzNWYxNTc4MjU2NjA0ZjFiNGQ0NTE2ODYiLCJleHAiOiIxNTQ0NDY1MDk3ODczIn0.SxFtZ7dqp9KsUSn1uTXhWis8Il8Hig8mwLANGU3cXhY"}'
```
Comment (kkelly51) [inline content/auth.md]: No, content type on this example request. On purpose?

File: content/elasticsearch.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Since reindex can be a long running operation, the `reindex-status` endpoint mus
 #### Example request
 
 ```curl
-$ curl --request POST https://example.com/v1/elasticsearch/reindex --header 'Authorization: Bearer ReplaceWithTheToken'
+$ curl --request POST https://example.com/elasticsearch/reindex --header 'Authorization: Bearer ReplaceWithTheToken'
```
Comment (kkelly51) [inline content/elasticsearch.md]: no content type

Comment (laurenfrederick) [reply content/elasticsearch.md]: So this one you don't send any data to

File: content/reconciliation-reports.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 ```endpoint
-GET /v1/reconciliationReports
+GET /reconciliationReports
 ```
 
 #### Example request
 
 ```curl
-$ curl https://example.com/v1/reconciliationReports --header 'Authorization: Bearer ReplaceWithTheToken'
+$ curl https://example.com/reconciliationReports --header 'Authorization: Bearer ReplaceWithTheToken'
```
[the remark this answers is not in the document]
Comment (laurenfrederick) [reply content/reconciliation-reports.md]: This one we're not sending any data to.

File: content/auth.md (hunk 2 above)
Comment (laurenfrederick) [reply content/auth.md]: Fixed, thanks!

**Pull request merged**